lib/router/selectedProduct.js

```diff
@@ -1,12 +1,12 @@
-FlowRouter.route('/ProductsExample', {
+FlowRouter.route('/SelectedProduct', {
     // Render the 'page'
     action: function(params, queryParams) {
       BlazeLayout.render('layout', {
         header:'header',
         body: 'main',
-        main: "productsExample"
+        main: "selectedProduct"
       });
     },
 
-    name: "productsExample" // optional
-});
+    name: "selectedProduct" // optional
+});
```

Comment:
Github doesn't like it when there isn't an empty line at the ends of files for some reason so I just put one in there to make it not show this since there probably is a reason for it. I'm pretty sure that my atom does it for me anyways.